civicrm.user.inc

```diff
@@ -11,6 +11,18 @@ use Drupal\Core\Session\AccountInterface;
  * Implements hook_user_login().
  */
 function civicrm_user_login(AccountInterface $account) {
+  $civicrm = \Drupal::service('civicrm');
+  $civicrm->initialize();
```

Comment:
This might not be necessary, but the other functions  do it, so I figured it was safer. I didn't want to create a new wrapper function in `src/Civicrm.php` just for `getSession` or equivalent (as was done for the synchronize function)